packages/SwingSet/test/vat-admin/terminate/vat-dude-terminate.js

```diff
@@ -16,6 +16,14 @@ export function buildRootObject(vatPowers) {
       return makePromiseKit().promise; // never fires
     },
 
+    dieHappy(completion) {
+      vatPowers.exitVat(completion);
```

Comment:
Hm, we need a test to confirm that syscalls made just before and after the `exitVat` are not reverted. That means sending messages to some external vat (probably the bootstrap) and making sure they arrive. I don't remember which `vat-*.js` is used by which test, so I don't know whether to recommend adding these messages here (and updating the test to expect them), or if this test is also exercising a bunch of detailed "what happens to outstanding inbound messages" behavior, in which case it might be easier to write a new test just for this one check. But either way we should demonstrate that `exitVat`/`exitVatWithFailure` does not revert the crank's syscalls.